proposals/0001-content-steering.md

```diff
@@ -0,0 +1,98 @@
+
+The Content Steering manifest is provided as a URI adjacent to these tags, with the expected response to be an object which generally looks like this:
+
+```json
+{
+ "VERSION": 1,
+ "TTL": 300,
+ "RELOAD-URI": "https://steeringservice.com/app/instance12345?session=abc",
+ "CDN-PRIORITY": ["beta","alpha"]
+}
+```
+This manifest provides further information about how frequently to refresh the steering manifest `TTL` in seconds, where to reload the steering manifest `RELOAD-URI` (optional), and the `CDN-PRIORITY` a priority-ordered array, which in this case is a descriptive term to represent **both** HLS and DASH implementations, which are represented as `PATHWAY-PRIORITY` and `SERVICE-LOCATION-PRIORITY` respectively.
+
+With this feedback from the content steering server, the player can then follow the prescribed content steering logic outlined in each spec to determine the where to fetch segments and when and where to fetch the next steering manifest.
+
+An example of how a typical interaction with the player, steering server and multiple CDNs is provided from the DASH IF spec.
+
+![Example DASH Content Steering Flow Diagram](./ContentSteering/DASH-Content-Steering-Flow.png)
+
+For more details on how this accomplished for each format, see the appropriate specification:
+* [HLS Content Steering Specifiation](https://developer.apple.com/streaming/HLSContentSteeringSpecification.pdf)
+* [DASH Content Steering Specification](https://dashif.org/docs/DASH-IF-CTS-00XX-Content-Steering-Community-Review.pdf)
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+To implement this feature in video.js, there will be 4 main components.
+* Parse `#EXT-X-CONTENT-STEERING` tags and associated properties in the m3u8-parser and expose them in the manifest object.
+* Parse `<ContentSteering>` tags and associated properties in the mpd-parser and expose them as well.
+* Add the ability to process the associated tags in VHS and request and refresh the content steering manifest.
+  - This would likely occur at the playlist-loader level as to more easily separate the DASH and HLS specific logic.
+* Process the content steering manifest and apply steering logic to both DASH and HLS content.
+  - This logic can be applied in common areas where applicable, but should be pushed up into the respective loaders when required.
+
+Both specifications (linked above) enumerate detailed steps on the required logic for handling both the steering manifest, new tags and associated properties. This logic must be implemented in a way specific to each format when necessary. For example, if required, [HLSv2 7.2 has specific requirements for pathway cloning](https://datatracker.ietf.org/doc/draft-pantos-hls-rfc8216bis/).
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+* This may not be a useful feature for *all* users of video.js.
```

Comment:
nit: I think adding "noticeable" works well here too:
```suggestion
* This may not be a useful or noticeable feature for *all* users of video.js.
```